frontend/src/scenes/surveys/Survey.tsx

```diff
@@ -96,29 +105,84 @@ export function SurveyForm({ id }: { id: string }): JSX.Element {
+                                            placeholder="https://posthog.com"
+                                        />
+                                    </Field>
+                                )}
+                                <Field name="description" label="Question description (optional)">
+                                    <LemonTextArea value={question.description || ''} />
+                                </Field>
+                                {question.type === SurveyQuestionType.Rating && (
+                                    <div className="flex flex-col gap-2">
+                                        <div className="flex flex-row gap-4">
+                                            <Field name="display" label="Display type" className="max-w-100">
+                                                <LemonSelect
+                                                    options={[
+                                                        { label: 'Number', value: 'number' },
+                                                        { label: 'Emoji', value: 'emoji' },
+                                                    ]}
+                                                />
+                                            </Field>
+                                            <Field name="scale" label="Scale" className="max-w-100">
+                                                <LemonSelect
+                                                    options={[
+                                                        ...((question as SurveyQuestionRating).display === 'emoji'
+                                                            ? [{ label: '1 - 3', value: 3 }]
+                                                            : []),
+                                                        { label: '1 - 5', value: 5 },
+                                                        ...((question as SurveyQuestionRating).display === 'number'
+                                                            ? [{ label: '1 - 10', value: 10 }]
+                                                            : []),
+                                                    ]}
+                                                />
+                                            </Field>
+                                        </div>
+                                        <div className="flex flex-row gap-4">
+                                            <Field
+                                                name="lowerBoundLabel"
+                                                label="Lower bound label"
+                                                className="max-w-150"
+                                            >
+                                                <LemonInput
+                                                    value={(question as SurveyQuestionRating).lowerBoundLabel || ''}
```

Comment:
ok I've got some pretty sus typing here, because survey questions have different types, open to suggestions 😅

Reply:
Take a look now, should be less sus ;)

Reply:
Nice